speed-up-tests/cloudbuild.py.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run Python tests, lint, etc.
+steps:
+  - id: flask_test
+    name: python:3.7
+    entrypoint: /bin/sh
+    waitFor: ['-']
+    args:
+      - -c
+      - |
+        ./run_test.sh -p
+
+  - id: package_js
+    name: gcr.io/numom-57642/test5:latest
+    entrypoint: /bin/bash
+    waitFor: ['-']
+    args:
+      - -c
+      - |
+        pwd
+        ls -l
```

Comment:
why are we running pwd and ls?

Reply:
sorry that was just for testing, i was having an issue where I couldnt see a file inside the container. Fixed.